packages/tailwindcss/package.json

```diff
@@ -35,6 +35,10 @@
       "require": "./src/compat/flatten-color-palette.cts",
       "import": "./src/compat/flatten-color-palette.ts"
     },
+    "./lib/util/flattenColorPalette.js": {
+      "require": "./src/compat/flatten-color-palette.cts",
+      "import": "./src/compat/flatten-color-palette.ts"
+    },
```

Comment:
You need to add this to `publishConfig` too

Reply:
Oh one day I will remember... 😢